readme.md

```diff
@@ -226,7 +226,8 @@ You can execute the plugin to validate the api files without breaking the build
 | excludeResources | Collection of String | | _obsoleted_ IGNORED paths in the API to exclude from the validation for all files. `x-ignore-rules` should be used instead. |
 | skipOnErrors | boolean | false | Parameter to avoid maven fail in case of validation error.                                                                  |
 | outputTypes | OutputType | CONSOLE | Output processors. The value can be: CONSOLE, JUNIT, LOG4J, NONE                                                            |
-| outputDir | File | target/ | Output directory for the validation report file (when outputType writes to a file)                                          |
+| outputDir | File | ${project.build.directory} | Output directory for the validation report file (when outputType writes to a file)                                          |
+| outputFile | File | outputDir/validationReport.json | Output file for JSON validation report. OutputDir and a default filename will be used when absent. |
```

Comment:
Also document `${rest-guide-validator.outputDir}/validationReport.json` here then, to clarify that `outputDir` won't be actual directory name.

While you're at it, could you also add a column listing the user property name for each parameter?

There's actually a way to generate a website based on the Mojo's annotations (like the checkstyle one), but it's more work than it's worth.

Maybe we should rename to jsonOutputFile bc multi-output is supported?